metatron-app/metatron-discovery
# #3736 Modify subtotal field width adjustment function

### Description

Modify subtotal field width adjustment function

**Related Issue** : #3736 


### How Has This Been Tested?

1. Check that the width of the subtotal field in the grid is adjustable.
2. Check if the number format is normally applied.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
